binance/lib/utils.py

```diff
@@ -13,7 +13,7 @@ def cleanNoneValue(d) -> dict:
 
 
 def check_required_parameter(value, name):
-    if not value:
+    if not value and value != 0:
```

Comment:
Can you give a case for when `and value != 0` is necessary?

Reply:
When fiat endpoints' transactionType = 0, it will be treated as falsy value and unable to pass the verification.